avaje-config/src/main/java/io/avaje/config/ConfigParsers.java

```diff
@@ -3,12 +3,26 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Map.Entry;
 import java.util.Set;
 
-/**
- * Holds the non-properties ConfigParsers.
- */
-final class Parsers {
+public interface ConfigParsers {
+
+  /** Return the ConfigParser for the given extension. */
+  ConfigParser get(String extension);
```

Comment:
Nullable return.

```java
@Nullable ConfigParser get...
```

If we are not using JSpecify then it has to be javadoc and I would do something like:

```java
/**
 * Return the ConfigParser for the given extension.
 * @return parser or <code>null</code>.
 */
/* @Nullable */ ConfigParser getOrNull(String extension);
```